redisearch/autocomplete.go

```diff
@@ -62,49 +65,85 @@ func (a *Autocompleter) AddTerms(terms ...Suggestion) error {
 	return nil
 }
 
+// AddTerms pushes new term suggestions to the index
+func (a *Autocompleter) DeleteTerms(terms ...Suggestion) error {
+	conn := a.pool.Get()
+	defer conn.Close()
+
+	i := 0
+	for _, term := range terms {
+
+		args := redis.Args{a.name, term.Term}
+		if err := conn.Send("FT.SUGDEL", args...); err != nil {
+			return err
+		}
+		i++
+	}
+	if err := conn.Flush(); err != nil {
+		return err
+	}
+	for i > 0 {
+		if _, err := conn.Receive(); err != nil {
+			return err
+		}
+		i--
+	}
+	return nil
+}
+
+// AddTerms pushes new term suggestions to the index
+func (a *Autocompleter) Length() (len int64, err error) {
+	conn := a.pool.Get()
+	defer conn.Close()
+	len, err = redis.Int64(conn.Do("FT.SUGLEN", a.name))
+	return
+}
+
 // Suggest gets completion suggestions from the Autocompleter dictionary to the given prefix.
 // If fuzzy is set, we also complete for prefixes that are in 1 Levenshten distance from the
 // given prefix
 //
 // Deprecated: Please use SuggestOpts() instead
-func (a *Autocompleter) Suggest(prefix string, num int, fuzzy bool) ([]Suggestion, error) {
+func (a *Autocompleter) Suggest(prefix string, num int, fuzzy bool) (ret []Suggestion, err error) {
```

Comment:
Since this is deprecated no real need to patch it up, but as you already have...